components/dashboard/src/projects/NewProject.tsx

```diff
@@ -263,6 +273,8 @@ export default function NewProject() {
                     </span>
                     <br/>
                     <button className="mt-6" onClick={() => reconfigure()}>Authorize Provider</button>
+                    <br/>
+                    <button className="mt-6" onClick={() => setShowGitProviders(true)}>Select Git Provider</button>
```

Comment:
Let's go with the plain link below. 👍🏻 

Redesign is somewhat in conflict with the update to the design in general.

Reply:
DEAL 🤝

Reply:
"Unfortunately" this didn't work out, so I "had" to rewrite some parts of the the initial page in order to show the "No Access" within the component (the one with the dropdown.) Otherwise it's hard to decide what to render when "gitlab.com" is selected. Also if you are connected with gitlab.com, this will be the new default. 

Also, there is a new state: "loading..." ;-)